src/azure-cli/requirements.py3.windows.txt

```
@@ -125,8 +125,8 @@ PyGithub==1.55
 PyJWT==2.4.0
 PyMySQL==1.0.2
 PyNaCl==1.5.0
+pyOpenSSL==22.1.0
 pyodbc==4.0.35
```

```suggestion
pyodbc==4.0.35
pyOpenSSL==22.1.0
```
It should be alphabetical order.